src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -35,13 +35,74 @@ <h5><span jhiTranslate="artemisApp.studentExamDetail.student">Student</span></h5
 
     <div class="mb-3">
         <h5><span jhiTranslate="artemisApp.studentExams.workingTime" [ngbTooltip]="getWorkingTimeToolTip()">Working time</span></h5>
-        <form class="d-flex" [formGroup]="workingTimeForm" (ngSubmit)="saveWorkingTime()">
-            <div class="me-2" [ngbTooltip]="getWorkingTimeToolTip()">
-                <input formControlName="hours" type="number" style="width: 60px; height: 35px" class="text-center" min="0" step="1" required /> h
-                <input formControlName="minutes" type="number" style="width: 60px; height: 35px" class="text-center" min="0" max="59" step="1" required /> min
-                <input formControlName="seconds" type="number" style="width: 60px; height: 35px" class="text-center" min="0" max="59" step="1" required /> s
+        <form #form="ngForm" (ngSubmit)="saveWorkingTime()">
+            <div class="mb-1" style="width: fit-content" [ngbTooltip]="getWorkingTimeToolTip()">
+                <div class="input-group mb-1 d-flex justify-content-between align-items-center">
+                    <label class="me-4" jhiTranslate="artemisApp.studentExams.setWorkingTime">Absolute:</label>
+                    <input
+                        [(ngModel)]="workingTimeFormValues.hours"
+                        (ngModelChange)="updateWorkingTimePercent()"
+                        [disabled]="isFormDisabled()"
+                        id="hours"
+                        name="hours"
+                        type="number"
+                        class="text-center form-control"
+                        style="width: 100px"
```

Comment:
We should avoid inline styles where possible. Especially as you're repeating this style for the other inputs, a class should be used here.

Reply:
Good point, done in 9db1dc851017457efe4d6d88648e9ac377202032.